.gitattributes

```diff
@@ -0,0 +1,2 @@
+spec/fixtures/vcr_cassettes -diff
```

Comment:
Maybe that is so, or maybe it's because the PR was started before adding this.  It may be that the PR is based on `merge` rather than `diff` data, maybe the next PR will be more telling.